src/pipelines/aws/aws-pipeline.ts

```diff
@@ -158,11 +158,20 @@ export default class AWSPipeline implements Pipeline {
       '$HEIGHT',
       targetResolution.height.toString()
     );
-    settingsStr = this.stringReplacement(
-      settingsStr,
-      '$BITRATE',
-      targetBitrate.toString()
-    );
+    if (settingsStr.includes('$BITRATE')) {
+      settingsStr = this.stringReplacement(
+        settingsStr,
+        '$BITRATE',
+        targetBitrate.toString()
+      );
+    }
+    if (settingsStr.includes('$QVBRQUALITYLEVEL')) {
+      settingsStr = this.stringReplacement(
+        settingsStr,
+        '$QVBRQUALITYLEVEL',
```

Comment:
Note that if we allow arbitrary variables (see above), we might want to support writing variables in the template in a way that for instance the case where we have both a variable `OUT` and `OUTPUT` would be handled correctly. We could for instance write the variables in the template like `${OUT}`.